ros_buildfarm/templates/snippet/publisher_publish-over-ssh.xml.em

```diff
@@ -21,6 +21,7 @@
                 <execTimeout>120000</execTimeout>
                 <usePty>false</usePty>
                 <useAgentForwarding>false</useAgentForwarding>
+                <useSftpForExec>false</useSftpForExec>
```

Comment:
The updated publish-over-ssh plugin includes support for exec via the
sftp protocol which should be disabled for our jobs and is by default.